src/pages/termTime.tsx

```diff
@@ -0,0 +1,89 @@
+            <>
+                <ErrorSummary errors={errors} />
+                <div className={`govuk-form-group ${errors.length > 0 ? 'govuk-form-group--error' : ''}`}>
+                    <fieldset className="govuk-fieldset" aria-describedby="term-time-hint">
+                        <legend className="govuk-fieldset__legend govuk-fieldset__legend--l">
+                            <h1 className="govuk-fieldset__heading">Is this ticket only valid during term times?</h1>
+                        </legend>
+                        <span className="govuk-hint" id="term-time-hint">
+                            Term dates should be specified in route and timetable data.
+                        </span>
+                        <FormElementWrapper errors={errors} errorId="term-time-yes" errorClass="govuk-radios--error">
+                            <div className="govuk-radios" id="term-time-radios">
+                                <div className="govuk-radios__item">
+                                    <input
+                                        className={`govuk-radios__input ${
+                                            errors.length > 0 ? 'govuk-input--error' : ''
+                                        } `}
+                                        id="term-time-yes"
+                                        name="termTime"
+                                        type="radio"
+                                        value="Yes"
+                                    />
+                                    <label className="govuk-label govuk-radios__label" htmlFor="term-time-yes">
+                                        Yes
+                                    </label>
+                                </div>
+                                <div className="govuk-radios__item">
+                                    <input
+                                        className={`govuk-radios__input ${
+                                            errors.length > 0 ? 'govuk-input--error' : ''
+                                        } `}
+                                        id="term-time-no"
+                                        name="termTime"
+                                        type="radio"
+                                        value="No"
+                                    />
+                                    <label className="govuk-label govuk-radios__label" htmlFor="term-time-no">
+                                        No
+                                    </label>
+                                </div>
+                            </div>
+                        </FormElementWrapper>
+                    </fieldset>
+                </div>
+                <input type="submit" value="Continue" id="continue-button" className="govuk-button" />
+            </>
+        </CsrfForm>
+    </TwoThirdsLayout>
+);
+
+const isTermTimeAttributeWithErrors = (
```

Comment:
is this the right place for this? we have the typeguard file where the rest of these live.